tests/arp/test_wr_arp.py

```diff
@@ -98,6 +99,7 @@ def setupFerret(self, duthost, ptfhost):
             '''
         )
         assert len(result['stderr_lines']) == 0, 'Could not obtain DIP'
```

Comment:
Please remove the redundant assert above.

Reply:
I'm not sure that this is redundant, it looks like it's checking for the command failing outright rather than just not returning anything. Is it possible for `stdout` to be contain content when `stderr_lines` also contains content?

Reply:
I suspect the return code will be zero in this case. If it is, the module runner will throw and so the assert is redundant in this case.

Reply:
Ok sounds good, will fix.